solutions/usaco-494.mdx

```diff
@@ -0,0 +1,119 @@
+---
+id: usaco-494
+source: Old Gold
+title: Guard Mark
+author: Ryan Chou
+---
+ 
+[Official Analysis](http://www.usaco.org/current/data/sol_guard_gold.html)
+
+## Explanation
+
+Note that since it might not always be optimal to keep the strongest/heaviest cow at the bottom of the stack, we have to go through every single possible subset, which is feasible with $N \leq 20$.
+
+Here's an example of a counter case:
+```
+4 6
+1 1000 4
+1 1 5
+3 2 3
+3 3 3
+```
+
+Let's go through a dynamic programming approach.
```

Comment:
something like
"hm bounds are 20"
"99% of the time that's bitmask dp so let's try that & see what happens"

Reply:
ok sure